core/app/models/spree/order.rb

```diff
@@ -671,13 +672,13 @@ def add_store_credit_payments
 
     def covered_by_store_credit?
       return false unless user
-      user.available_store_credit_total(currency: currency) >= total
+      user.available_store_credit_total(currency: currency, force_reload: false) >= total
```

Comment:
I think we're changing the behavior here, as it was `reload`ing previously... can you explain the reason for this?